applications/luci-app-cloudflared/htdocs/luci-static/resources/view/cloudflared/log.js

```diff
@@ -0,0 +1,128 @@
+/* SPDX-License-Identifier: GPL-3.0-only
+ *
+ * Copyright (C) 2024 Hilman Maulana <hilman0.0maulana@gmail.com>
+ */
+
+'use strict';
+'require fs';
+'require ui';
+'require view';
+'require poll';
+
+function formatDate(dateTime) {
```

Comment:
the function converts a UNIX timestamp to a date and time in the ISO format but with a space instead of `T`.
First of all: You can make it a one liner:  `new Date(dateTime).toISOString().replaceAll('T', ' ')`

But why not to use a local date format? E.g. `new Date(dateTime).toLocaleString()`